messages/setup.json

```diff
@@ -31,5 +31,9 @@
     "android:reqs:platformapi:unfulfilledMessage" : "Install at least one Android API package (%s - %s).",
     "android:reqs:emulatorimages:title" : "Checking SDK Platform Emulator Images",
     "android:reqs:emulatorimages:fulfilledMessage" : "Android emulator image (%s) was detected.",
-    "android:reqs:emulatorimages:unfulfilledMessage" : "Install at least one of these emulator images: %s"
+    "android:reqs:emulatorimages:unfulfilledMessage" : "Install at least one of these emulator images: %s",
+   
+    "common:reqs:serverplugin:title" : "Checking sfdx LWC server plugin",
```

Comment:
My preference (for this and the two below) would be to refer to the plugin explicitly by name.  I.e. `@salesforce/lwc-dev-server`.  That way, people have a clear reference to the explicit dependency.